src/SFA.DAS.Reservations.Web/Views/Reservations/Review.cshtml

```diff
@@ -51,7 +51,7 @@
                             </span>
                         </summary>
                         <div class="govuk-details__text">
-                            You can change the course when adding an apprentice's details, without affecting your funding.
+                            You can change the course when adding an apprentice’s details.
```

Comment:
Can we check its the correct apostrophe symbol @ChughAkshay @jamesmking

Reply:
I would always go for the vertical apostrophe `'`